### PR TITLE
feat: add env example and tx folder

### LINKED ISSUE #21: Create a folder to test governance transactions

### PR DESCRIPTION
**Pull request by pakim249CAL**
Fixes #21

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NETWORK=ethereum-mainnet
+TRANSACTION_NAME=test
```
Comment (MerlinEgalite) [inline .env.example:2]: can't we use the filename instead of an env variable?

Comment (Tristan22400) [reply .env.example:2]: I agree that it is more simple.

**Review by Tristan22400 — APPROVED** [review]

**Review by Rubilmax — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (pakim249CAL) [reply .env.example:2]: I'm not sure I understand your suggestion @MerlinEgalite

Comment (MerlinEgalite) [reply .env.example:2]: forget about that

Comment (Rubilmax) [reply .env.example:2]: Perhaps shouldn't I have mixed my suggestion with this discussion. I see you upvoted my comment Patrick, did you forget about implementing the suggested change?

**Review by makcandrov — APPROVED** [review]

**Review by Rubilmax — APPROVED** [review]

**Review by Tristan22400 — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Pull request merged**